### DISCUSSION TITLE
<Can i able to make github pages for different repos that are present >

### DISCUSSION THREAD

**Discussion by Harish-2003** [community/community#123779] [Other Feature Feedback, Questions, & Ideas]
Hi , i want to make websites for different repos that are present , how can i do it. is it possible to make use of github pages for multiple repos that are present

**Comment by naveenalla3000** [comment]
@Harish-2003 simple solution is every repo you want web-site add and host on git-hub pages.

**Reply by Harish-2003** [reply]
How can i do that i can do one repo by naming it as username.github.io how can I able to do it for multiple repos

**Reply by naveenalla3000** [reply]
Navigate to repo that you want to host and go to settings of that repo and click on page and select the branch and deploy it.
